Automattic/jetpack
# [WordAds] Add new US states to do not sell privacy check and update copy

#### Changes proposed in this Pull Request:
This PR adds new US states to the "Do Not Sell My Personal Information" privacy check and updates the WordAds Privacy & Consent settings copy and Do Not Sell modal copy to serve generic wordings that are applicable to all the other included US states and not just California.

#### Other information:


#### Jetpack product discussion
pedfti-fp-p2

#### Does this pull request change what data or activity we track or use?
No



#### Testing instructions:
- Create or use an existing Jetpack site.
- Enable targeted advertising to site visitors in all US states. This can be done via the wp-admin `Jetpack -> Settings -> Traffic -> Ads`
- Notice that the copy has now been updated to the new privacy copy from legal team.
 <img width="700" alt="copy" src="https://user-images.githubusercontent.com/9039613/197712735-52e83d9a-42a4-414b-99e0-c1ff507104d2.png">
- Clicking on the tooltip should display updated support copy

Previous copy
<img width="600" alt="before" src="https://user-images.githubusercontent.com/9039613/197956258-ca56a903-52a0-497c-a0af-455c580aa603.png">
New copy with updated link to support page on atomic site. 
<img width="990" alt="link" src="https://user-images.githubusercontent.com/9039613/199886215-29ed798c-a426-406f-9d38-2f1c78ae751f.png">

- Under "Appearance > Themes", click on "customize" on your theme.
- Place the Do Not Sell widget or add the shortcode `[privacy-do-not-sell-link]` (previously `[ccpa-do-not-sell-link]`) to the footer of the site's page.
- Using a VPN set to any of the US states - California, Colorado, Connecticut, Utah, or Virginia, visit your Jetpack site
- Scroll to the footer section you should see a "Do Not Sell My Personal Information" link
-Click on the "Do Not Sell My Personal Information" link
<img width="1231" alt="privacy link" src="https://user-images.githubusercontent.com/9039613/197713267-b681e894-3774-4cc9-b506-195b80896fee.png">

- A modal should appear and the copy on the modal should now be updated to the new privacy copy from legal.
<img width="350" alt="updated modal copy" src="https://user-images.githubusercontent.com/9039613/197712168-c26e0b55-5412-4a51-8403-6f2ae8e37520.png">
